assets/pages/digital-subscription-landing/components/digitalSubscriptionLandingHeader.jsx

```diff
@@ -18,7 +18,7 @@ import SvgChevron from 'components/svgs/chevron';
 import { CirclesLeft, CirclesRight } from 'components/svgs/digitalSubscriptionLandingHeaderCircles';
 import AnchorButton from 'components/button/anchorButton';
 import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
-import { displayPrice, sendTrackingEventsOnClick, type SubscriptionProduct } from 'helpers/subscriptions';
+import { sendTrackingEventsOnClick, type SubscriptionProduct } from 'helpers/subscriptions';
```

Comment:
Was this meant to be imported?

Reply:
It was being used in the old header copy but updated copy doesn't require `displayPrice`